From 199c86591edc7e82b92903efecadc4f69ea63370 Mon Sep 17 00:00:00 2001
From: Johannes Morgenroth <jm@m-network.de>
Date: Tue, 1 Jan 2019 11:25:50 +0100
Subject: [PATCH] Add operator!=() to Bundle::block_elem

Alternative implementations of libstdcpp as uclibc++ use it in
the algorithm implemenetations.
---
 ibrdtn/data/Bundle.h | 3 +++
 1 file changed, 3 insertions(+)

--- a/ibrdtn/data/Bundle.h
+++ b/ibrdtn/data/Bundle.h
@@ -69,6 +69,9 @@ namespace dtn
 				bool operator==(const dtn::data::block_t &type) const {
 					return (**this) == type;
 				}
+				bool operator!=(const dtn::data::block_t &type) const {
+					return !((**this) == type);
+				}
 			};
 
 			typedef std::list<block_elem> block_list;
